django/forms/forms.py

```diff
@@ -210,17 +210,20 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
                 if css_classes:
                     html_class_attr = ' class="%s"' % css_classes
 
+                extra_attrs = {'aria-describedby': f'{bf.id_for_label}_helptext'}
                 if errors_on_separate_row and bf_errors:
                     output.append(error_row % str(bf_errors))
+                    extra_attrs['aria-describedby'] += f' {bf.id_for_label}_errorlist'
```

Comment:
I think it would be more correct for the error description to be first, and help text second, as in Django’s default rendering the error messages appear before the field while the help text appears after.